webrtc.html

```diff
@@ -83,6 +83,11 @@
     <dfn>MediaStreamConstraints</dfn> are defined in [[!GETUSERMEDIA]].</p>
     <p>The term <dfn>Blob</dfn> is defined in [[!FILEAPI]].</p>
     <p>The term <dfn>media description</dfn> is defined in [[!RFC4566]].</p>
+    <p>When referring to exceptions, the terms <dfn><a
+    href="https://www.w3.org/TR/WebIDL-1/#dfn-throw">throw</a></dfn> and
```

Comment:
My concern was that this links to "latest published version", and there's no hard guarantee that a future published version won't remove the "dfn"s being linked to.